### PR TITLE
implemetation of notify btn

### PR DESCRIPTION
**Pull request by Laurajcb**
added function to verify email and implement the logic to send data via email, got some errors with the ref  properties on the form and also have some parameter thats has implicit type

### CODE REVIEW

File: src/components/UnderConstruction.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useRef } from "react";
 import Image from "next/image";
 import { FaWhatsapp } from "react-icons/fa";
 import Link from 'next/link';
 import styles from "@/styles/components/UnderConstruction.module.css";
+import emailjs from '@emailjs/browser';
```
Comment (jacastrorug) [inline src/components/UnderConstruction.tsx:6]: @emailjs is a yarn dependency?

**Comment by jacastrorug** [comment]
You cannot do a pull request directly to main, please change the target to develop

File: src/components/UnderConstruction.tsx (hunk 1 above)
Comment (Laurajcb) [reply src/components/UnderConstruction.tsx:6]: {
  "dependencies": {
    "@emailjs/browser": "^3.10.0"
  }
}
yeah you can see at the package.json

Comment (Laurajcb) [reply src/components/UnderConstruction.tsx:6]: oh I see what you said let me fix it

Comment (Laurajcb) [reply src/components/UnderConstruction.tsx:6]: I already fix the dependency but i still have errors , can you check please?

**Comment by jacastrorug** [comment]
Closed because of the target, should be development no main

**Pull request closed without merging**